src/pages/ServiceDetails/ServiceInfo/IstioObjectDetails/DestinationRuleDetail.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import { Col, Icon, Row, Table } from 'patternfly-react';
+import {
+  checkForPath,
+  globalChecks,
+  highestSeverity,
+  severityToColor,
+  severityToIconName,
+  validationToSeverity,
+  DestinationRule,
+  Subset
+} from '../../../../types/ServiceInfo';
+import { ObjectValidation } from '../../../../types/IstioObjects';
+import LocalTime from '../../../../components/Time/LocalTime';
+import DetailObject from '../../../../components/Details/DetailObject';
+import * as resolve from 'table-resolver';
+import Label from '../../../../components/Label/Label';
+
+interface DestinationRuleProps {
+  namespace: string;
+  destinationRule: DestinationRule;
+  validations: { [key: string]: ObjectValidation };
+}
+
+class DestinationRuleDetail extends React.Component<DestinationRuleProps> {
+  constructor(props: DestinationRuleProps) {
+    super(props);
+  }
+
+  validation(destinationRule: DestinationRule): ObjectValidation | undefined {
+    if (this.props.validations && this.props.validations[destinationRule.name]) {
+      return this.props.validations[destinationRule.name];
+    }
+    return undefined;
+  }
+
+  globalStatus(rule: DestinationRule) {
+    let validation = this.validation(rule);
+    if (!validation) {
+      return '';
+    }
+    let checks = globalChecks(validation);
+    let severity = validationToSeverity(validation);
+    let iconName = severityToIconName(severity);
+    let color = severityToColor(severity);
+    let message = checks.map(check => check.message).join(',');
+
+    if (!message.length) {
+      if (validation && !validation.valid) {
+        message = 'Not all checks passed!';
+      }
+    }
+
+    if (message.length) {
+      return (
+        <div>
+          <p style={{ color: color }}>
+            <Icon type="pf" name={iconName} /> {message}
+          </p>
+        </div>
+      );
+    } else {
+      return '';
+    }
+  }
+
+  hostStatusMessage(destinationRule: DestinationRule) {
```

Comment:
not used?

Reply:
I had a question here...because...Have we got validations for this? I needed the change the method
```js
validation(destinationRule: DestinationRule): ObjectValidation | undefined {
    if (this.props.validations && this.props.validations[destinationRule.name]) {
      return this.props.validations[destinationRule.name];
    }
    return undefined;
  }
```

because this.props.validations[destinationRule.name] is undefined, the behaviour is diferent to virtualservices. I need to investigate about